### ISSUE TITLE
change picture problems

### ISSUE THREAD

**Issue by maplewizard** [shd101wyy/markdown-preview-enhanced#144]
I inserted a png picture into the markdown file. But when I modified the picture, the rendered page still present the old picture. Is this a bug?

**Comment by shd101wyy** [comment]
No it is not a bug. This happened because the image is cached by the browser. You can refresh your image by adding something like "?blabla" after the image URL.

I will add support for uncached image in next version.

**Comment by maplewizard** [comment]
@shd101wyy I changed the image name and change it back, it still not work. Is there any way to force refresh?

**Comment by shd101wyy** [comment]
Nope, there is no way to force refresh. You can just use the name that you changed to.

"test.png?1" 
"test.png?2"
...

**Comment by shd101wyy** [comment]
Check https://github.com/atom/markdown-preview/issues/207

**Comment by moriyas1984** [comment]
Excuse me cutting in.
You can reload ATOM by pressing Ctrl + Alt + R, I think it is the most easy solution about the problem at the moment. 
Of course you might already know...

**Comment by eugene** [comment]
I was thinking that it should be possible to optimally update images using a technique similar to one used in [markdown-preview-plus package](https://github.com/atom-community/markdown-preview-plus/) (where this issue is solved). As far as my understanding goes, it looks like every image in the document is registered and file system is watched for changes. Here is their [image-watch-helper.coffee](https://github.com/atom-community/markdown-preview-plus/blob/master/lib/image-watch-helper.coffee)
